core/templates/pages/exploration-editor-page/translation-tab/state-translation/state-translation.directive.ts

```diff
@@ -332,10 +383,59 @@ angular.module('oppia').directive('stateTranslation', [
             return {'border-left': '3px solid ' + color};
           };
 
-          $scope.getHtmlSummary = function(subtitledHtml) {
-            var htmlAsPlainText = $filter(
-              'formatRtePreview')(subtitledHtml.getHtml());
-            return htmlAsPlainText;
+          $scope.getSubtitledContentSummary = function(subtitledContent) {
+            if (subtitledContent instanceof SubtitledHtml) {
+              return $filter('formatRtePreview')(subtitledContent.getHtml());
+            } else if (subtitledContent instanceof SubtitledUnicode) {
+              return subtitledContent.getUnicode();
+            }
+          };
+
+          $scope.getInteractionCustomizationArgTranslatableContent = function(
+              customizationArgs: InteractionCustomizationArgs
+          ): {name: string, content: SubtitledUnicode|SubtitledHtml}[] {
+            const translatableContent = [];
```

Comment:
translatableContent**s** if it's a list? (Ditto for the function name?)

Reply:
Done